tests/shared/src/test/scala/scala/meta/tests/parsers/dotty/ExtensionMethodsSuite.scala

```diff
@@ -342,124 +342,6 @@ class ExtensionMethodsSuite extends BaseDottySuite {
     ))
   }
 
-  test("extension quasiquotes: triple, no tparams") {
```

Comment:
Moved to DottySuccessSuite in Scala 2, since it uses quasiquotes which are not yet supported